src/main/java/uk/gov/hmcts/ccd/domain/service/createcase/DefaultCreateCaseOperation.java

```diff
@@ -14,9 +14,7 @@
 import uk.gov.hmcts.ccd.data.user.UserRepository;
 import uk.gov.hmcts.ccd.domain.model.aggregated.IdamUser;
 import uk.gov.hmcts.ccd.domain.model.callbacks.AfterSubmitCallbackResponse;
-import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
-import uk.gov.hmcts.ccd.domain.model.definition.CaseEvent;
-import uk.gov.hmcts.ccd.domain.model.definition.CaseType;
+import uk.gov.hmcts.ccd.domain.model.definition.*;
```

Comment:
Not sure if checkstyle likes .*  organise imports

Reply:
agree with you Kiran. I did raise a ticket for this it's been in Kanban for a while but never gets picked up :)

Reply:
resolving conversation as it's nothing particular relate to this PR.